### DISCUSSION TITLE
student benefits program

### DISCUSSION THREAD

**Discussion by mza0110** [community/community#30161] [GitHub Education]
hello, my request was not approved to join the student benefits program of GitHub because im not in the location of my university Zhejiang university of technology due to summer vacation i can send you all the documents you need that proves that im a student at Zhejiang university of technology but im not in china at the moment, im in Dubai for the summer vacation does someone can help me with that

**Comment by melvincwng** [comment]
@mza0110 , I would suggest you to raise a ticket to GitHub Education Support over [here](https://support.github.com/contact/education).

They are the most relevant team to handle issues with regards to the GitHub Student Developer pack.

Describe your issue and provide them a few days (or weeks) to resolve your issue.

Hope this is helpful

**Reply by mza0110** [reply]
thank you very much

**Reply by melvincwng** [reply]
@mza0110 , pleasure to help!
